packages/switch/stories/index.stories.js

```diff
@@ -0,0 +1,90 @@
+import { storiesOf, html } from '@open-wc/demoing-storybook';
+import { LitElement } from '@lion/core';
+
+import { LocalizeMixin } from '@lion/localize';
+
+import '../lion-input-switch.js';
+import '@lion/form/lion-form.js';
+
+storiesOf('Switches|Switch', module)
+  .add(
+    'Default',
+    () => html`
+      <lion-input-switch></lion-input-switch>
+    `,
+  )
+  .add(
+    'All text slots',
+    () => html`
+      <lion-input-switch label="Label" help-text="Help text">
+        <div slot="after">After</div>
```

Comment:
remove the "after" slotted div

Reply:
I think the "all text slots" can be the default demo. It should never be used without a label anyway.